javascript/javascript2/week3/warmup1.js

```diff
@@ -0,0 +1 @@
+setTimeout(() => { console.log("Called after 2.5 seconds"); }, 2500);
```

Comment:
That will work but it is not very reusable and it is not a funcion.
Think about how could you write a funcion that will have a text as a parameter and then delay the console log? Something like this for example:
```
funcion delayLog(message) {
    setTimeout(() => { console.log(message); }, 2500);
}
```
Now how would you modify this so we can also decide how long we want to delay the logging